src/editor/components/seSpinInput.js

```diff
@@ -8,6 +8,10 @@ template.innerHTML = `
     margin: 5px 1px;
     padding: 3px;
   }
+  div.imginside {
+    --se-spin-input-width: var(--global-se-spin-input-width);
+    width: var(--se-spin-input-width);
```

Comment:
why do we need --se-spin-input-width and --global-se-spin-input-width ?? 
Also, you added the width parameter to the se-spin-input. 
I think we need to decide between defining a global variable and reuse it inside the component or adding a width attribute to se-spin-input.

Reply:
shall i use global variable or width attribute ?